webapp/tests/Controller/Jury/SubmissionControllerTest.php

```diff
@@ -0,0 +1,43 @@
+<?php declare(strict_types=1);
+
+namespace App\Tests\Controller\Jury;
+
+use App\DataFixtures\Test\SampleSubmissionsFixture;
+use App\Tests\BaseTest;
+use Generator;
+
+class SubmissionControllerTest extends BaseTest
+{
+    protected static $roles = ['jury'];
+    protected static $baseURL = '/jury/submissions';
+
+    /**
+     * Test that the basic building blocks of the index page are there.
+     */
+    public function testIndexBasic(): void
+    {
+        $this->verifyPageResponse('GET', static::$baseURL, 200);
+    }
+
+    /**
+     * Test the filtered views do throw server errors
+     *
+     * @dataProvider provideViews
+     */
+    public function testIndexViewFilter(string $filter, array $fixtures): void
+    {
+        if (!empty($fixtures)) {
+            static::$fixtures = $fixtures;
```

Comment:
This will not work. Call $this->loadFixture

Reply:
Should now be fixed with a foreach loop.